src/main/java/com/hackclub/hccore/commands/NickCommand.java

```diff
@@ -43,8 +43,9 @@ public boolean onCommand(CommandSender sender, Command cmd, String alias, String
         }
 
         this.plugin.getDataManager().getData(player).setNickname(newNickname);
-        sender.sendMessage(
-                ChatColor.GREEN + "Your nickname was set to " + ChatColor.AQUA + newNickname);
+        ChatColor nameColor = this.plugin.getDataManager().getData(player).getNameColor();
+        sender.sendMessage(ChatColor.GREEN + "Your nickname was set to "
+                + (nameColor != null ? nameColor : ChatColor.AQUA) + newNickname);
```

Comment:
PlayerData#getNameColor will never be null because PlayerData#setColor will automatically default to white if you attempt to set it to `null` ([see here](https://github.com/hackclub/HCCore/blob/49962239de1ba12373d2fa19bf07e0eaa2429334/src/main/java/com/hackclub/hccore/PlayerData.java#L116)). The null check can be removed.

Reply:
oh awesome, didn't realize it would do that - i'll remove the check and just set it to name color only